src/protocol/channels.ts

```diff
@@ -332,7 +334,7 @@ export type BrowserTypeLaunchPersistentContextOptions = {
   downloadsPath?: string,
   chromiumSandbox?: boolean,
   slowMo?: number,
-  noDefaultViewport?: boolean,
+  showUserInput?: boolean,
```

Comment:
- what happened to the noDefaultViewport?
- I would not want to add another API like `headless: false` that requires modifications of source for every debugging session.

Reply:
We are missing some doc checks these days.